.github/workflows/publish-to-pypi.yml

```diff
@@ -1,37 +1,37 @@
-name: Publish Python distributions to PyPI
+# name: Publish Python distributions to PyPI
 
-on:
-  push:
-    branches:
-      - main
+# on:
+#   push:
+#     branches:
+#       - main
 
-jobs:
-  build-n-publish:
-    name: Build and publish Python distributions to PyPI and TestPyPI
-    runs-on: ubuntu-18.04
+# jobs:
+#   build-n-publish:
+#     name: Build and publish Python distributions to PyPI and TestPyPI
+#     runs-on: ubuntu-18.04
 
-    steps:
-    - uses: actions/checkout@master
-    - name: Set up Python 3.8
-      uses: actions/setup-python@v1
-      with:
-        python-version: 3.8
+#     steps:
+#     - uses: actions/checkout@master
+#     - name: Set up Python 3.8
+#       uses: actions/setup-python@v1
+#       with:
+#         python-version: 3.8
 
-    - name: Install pypa/build
-      run: >-
-        python -m
-        pip install
-        build
-        --user
-    - name: Build a binary wheel and a source tarball
-      run: >-
-        python -m
-        build
-        --sdist
-        --wheel
-        --outdir dist/
-    - name: Publish distribution to PyPI
-      # if: startsWith(github.ref, 'refs/tags')
-      uses: pypa/gh-action-pypi-publish@v1.4.2
-      with:
-        password: ${{ secrets.PYPI_API_TOKEN }}
\ No newline at end of file
+#     - name: Install pypa/build
+#       run: >-
+#         python -m
+#         pip install
+#         build
+#         --user
+#     - name: Build a binary wheel and a source tarball
+#       run: >-
+#         python -m
+#         build
+#         --sdist
+#         --wheel
+#         --outdir dist/
+#     - name: Publish distribution to PyPI
+#       # if: startsWith(github.ref, 'refs/tags')
+#       uses: pypa/gh-action-pypi-publish@v1.4.2
+#       with:
+#         password: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
Can we just delete the file? You can always get it back w/ git later.

Reply:
Yes